Makefile

```diff
@@ -108,7 +108,9 @@ go/lint: $(BIN)/golangci-lint
 .PHONY: go/mod
 go/mod:
 	GO111MODULE=on go mod download
-	GO111MODULE=on go mod verify
+	# doesn't work for go workspace
+	# GO111MODULE=on go mod verify
+	go work sync
 	GO111MODULE=on go mod tidy
```

Comment:
I think you also need to run `go mod tidy` in folder `api/`

Reply:
I think so too.